### PR TITLE
Installing Code Coverage (CodeCov)

### LINKED ISSUE #74: Support collecting coverage when running tests

### PR DESCRIPTION
**Pull request by poush**
Fixes #74 

Installing Code Coverage, using CodeCov. 
@marco-c Once this application is integrated with the repo we can get the reports in PRs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,5 +8,7 @@ tools/nightly/*
 tools/geckodriver
 tools/chromedriver
 
+inconsistencies.csv
```
Comment (marco-c) [inline .gitignore]: Move this after `data_resized/`

Comment (marco-c) [reply .gitignore]: Actually, remove it altogether, since it's unrelated to this PR.

File: .coveragerc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[report]
+omit =
+    */data/*
```
Comment (marco-c) [inline .coveragerc]: Use `data/*`, since we know the directory is at the top level (same for the others).

**Comment by poush** [comment]
@marco-c Should I create an issue for adding `inconsistencies.csv` in the .gitignore ?

**Comment by marco-c** [comment]
You can also create a PR directly.

**Pull request merged**